test/core.spec.ts

```diff
@@ -43,12 +43,54 @@ describe('powerbi', function () {
             expect(iframes.length).toEqual(2);
         });
     });
+    
+    describe('get', function () {
+        it('if attempting to get a powerbi component on an element which was not embedded, throw an error', function () {
+            // Arrange
+            const component = $('<div></div>');
+            
+            // Act
+            const attemptGet = () => {
+                powerbi.get(component[0]);
+            };
+            
+            // Assert
+            expect(attemptGet).toThrowError(Error);
+        });
+        
+        it('calling get on element with embeded component returns the instance', function () {
+            // Arrange
+            const $element = $('<div powerbi-type="report" powerbi-embed-url="https://app.powerbi.com/reportEmbed?reportId=ABC123"></div>')
+                .appendTo('#powerbi-fixture');
+            
+            const componentInstance = powerbi.embed($element[0]);
```

Comment:
Just a good practice to use most restrictive type you can for the use case.  It results in safer code.  It's similar the principle of least privilege or why most languages default class variables to private instead of public.  In summary `const` is the default, and only when you expect the variable to be reassigned use `let` and only when you rely on variable hoisting / specific behavior do you use `var`.  If you use `let` everywhere developers won't know if you intended it to be reassigned or if it was accidental, but this way it becomes important information to others.